authorization/authorization.py

```diff
@@ -127,7 +127,7 @@ def message_passed_interception_filters(self, messageInfo):
                 message_passed_filters = False
 
         if self.IFList.getModel().getElementAt(i).split(":")[0] == "Response Body contains (regex)":
-            regex_string = self.IFList.getModel().getElementAt(i)[33:]
+            regex_string = self.IFList.getModel().getElementAt(i)[:33]
```

Comment:
Does this this on purpose? Can you elaborate why you did it?

Reply:
Apologies, this should not have been changed. Please reject.